internal/config/server_test.go

```diff
@@ -1457,7 +1457,7 @@ func TestServer_Opts(t *testing.T) {
 				Restart: false,
 			},
 			want: want{
-				want: make([]server.Option, 25),
+				want: make([]server.Option, 26),
```

Comment:
We should check it more strictly.... :thinking:

Reply:
I thinks so....

Reply:
@vdaas/set Can you please send the PR to fix the Opts function tests in internal/config in a separate PR, since the entire test code for this part is checked like this.